gdx/src/com/badlogic/gdx/utils/Pools.java

```diff
@@ -34,10 +37,37 @@ static public <T> Pool<T> get (Class<T> type, int max) {
 
 	/** Returns a new or existing pool for the specified type, stored in a Class to {@link Pool} map. The max size of the pool used
 	 * is 100. */
+	@Deprecated
 	static public <T> Pool<T> get (Class<T> type) {
 		return get(type, 100);
 	}
 
+	/** Returns an existing pool or null for the specified type, stored in a Class to {@link Pool} map. */
+	static public <T> Pool<T> getOrNull (Class<T> type) {
+		return typePools.get(type);
+	}
+
+	/** Returns a new or existing pool for the specified type, stored in a Class to {@link Pool} map. A new pool will be of type
+	 * {@link DefaultPool} */
+	static public <T> Pool<T> getOrDefault (Class<T> type, PoolSupplier<T> poolTypeSupplier) {
```

Comment:
Something about this seems strange to me. This doesn't act like Java 8's Map.getOrDefault(), since it takes a supplier rather than a default return value (which would be a `Pool<T>`). I'm not sure if there's a better name, though... `getOrCompute()` would probably be close to the behavior, but I'm not sure...

Reply:
Yeah, as mentioned in the other comment, I struggled with the method and method names. Unfortunatly, we can't seperate method signatures by generics, thats the reason of the weird name.
But yeah, I will further think on the `Pools` class and methods.

Reply:
For what it's worth, `computeIfAbsent` is the equivalent name.